tests/rptest/tests/openmessaging_benchmark_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import time
+
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.services.openmessaging_benchmark_worker import OpenMessaging_Benchmark_Worker, OpenMessaging_Benchmark
+
+
+class OpenBenchmarkTest(RedpandaTest):
+    def __init__(self, ctx):
+        self._ctx = ctx
+        super(OpenBenchmarkTest, self).__init__(test_context=ctx,
+                                                num_brokers=3)
+
+    @cluster(num_nodes=4)
+    def test_sleep(self):
+        worker = OpenMessaging_Benchmark_Worker(self.redpanda)
+
+        benchmark = OpenMessaging_Benchmark(self._ctx, self.redpanda)
```

Comment:
I'm having trouble understanding the different between `OpenMessaging_Benchmark_Worker` and `OpenMessaging_Benchmark`. It looks like they should be combined?